edxval/models.py

```diff
@@ -600,6 +600,13 @@ class TranscriptPreference(TimeStampedModel):
         blank=True,
     )
     preferred_languages = ListField(verbose_name='Preferred Languages', max_items=50, default=[], blank=True)
+    video_source_language = models.CharField(
+        verbose_name='Video Source Language',
+        max_length=20,
+        null=True,
+        blank=True,
+        help_text='This is the source language of a video.'
```

Comment:
can we also add that this for 3play for now.

Reply:
I agree, that'd be great.